imputation/ts_impute_test.go

```diff
@@ -55,12 +116,19 @@ func createOutput(ts TimeSeries, desiredValues []string, method string) TimeSeri
 				out[k] = median
 			}
 		}
+	case "spline1":
+		if len(ts) > 2 {
+			series := linearInterpolate(desiredValues[0])
+			for _, k := range desiredValues {
+				out[k] = series[k]
+			}
+		}
 	}
 	return (out)
 }
 
 func TestFillNA(t *testing.T) {
-	methods := []string{"mean", "zero", "median"}
+	methods := []string{"mean", "zero", "median", "spline1"}
```

Comment:
"spline1" is implemented by Interpolate, not FillNA, so it should be tested in a function with a name like `TestInterpolate`.

Reply:
Right! I changed the testing structure. Now it'll be easy to add test values for quadratic and cubic ones. There is a structure (InterpolateValue) that I can easily add more to its values.